source/atlas-cli-env-variables.txt

```diff
@@ -98,4 +98,8 @@ The {+atlas-cli+} supports the following environment variables:
 
    * - ``MONGODB_ATLAS_REFRESH_TOKEN``
      - String that allows |service| to automatically request a new
-       access token when the current access token expires.
\ No newline at end of file
+       access token when the current access token expires.
+
+   * - ``MONGODB_ATLAS_TELEMETRY_ENABLED``
```

Comment:
I'm not too sure if we should document it but we allow users to set `DO_NOT_TRACK=1` to disable it as well

according to https://consoledonottrack.com/